index.html

```diff
@@ -0,0 +1,23 @@
+<!DOCTYPE html>
```

Comment:
Are you using this file for anything? If not it can be deleted

Reply:
It's not doing anything so I'll delete it